scipy/interpolate/fitpack2.py

```diff
@@ -932,6 +943,55 @@ def __call__(self, x, y, dx=0, dy=0, grid=True):
             z = z.reshape(shape)
         return z
 
+    def partial_derivative(self, nux, nuy):
+        """Construct a new spline representing a partial derivative of this
+        spline.
+
+        Parameters
+        ----------
+        nux, nuy : int
+            Orders of the derivative in x and y respectively. They must be
+            non-negative integers and less than the respective degree of the
+            original spline (self) in that direction (kx, ky).
+
+        Returns
+        -------
+        spline :
+            A new spline of degrees (kx - nux, ky - nuy) representing the
+            derivative of this spline.
+
+        Notes
+        -----
+        The returned spline is an instance of the class
+        ``_DerivedBivariateSpline`` that supports spline evaluation operations,
+        but is not directly constructed from original data.
+        """
+        if nux == 0 and nuy == 0:
+            return self
```

Comment:
Just thinking out loud: non-zero values create a copy , while this returns essentially a view. Could this be problematic? Cannot come up with a test case right now, so maybe it's FUD.